docker-apt.list

```diff
@@ -1 +1 @@
-deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian bullseye stable
+deb [signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian bullseye stable
```

Comment:
nice, didnt realize we didnt have to specify the arch here. makes multi-arch much easier to build.